### DISCUSSION TITLE
How to capture code-level span

### DISCUSSION THREAD

**Discussion by zhangjiabin1010** [open-telemetry/opentelemetry-java-instrumentation#11661] [Q&A] [answered]
such as the custom method that pinpoint can capture. If I want to capture this level of span, how should I do it? I hope to get guidance

**Comment by steverao** [comment, marked as answer]
Do you want to produce some extra spans beyond Java Agent for your own methods? If so, you can consider manual instrumentation for your own methods, there are some [API documents](https://opentelemetry.io/docs/languages/java/api/#api-components). BTW, there is an [example](https://github.com/open-telemetry/opentelemetry-java-examples/tree/main/javaagent) that show how to instrument your own methods by SDK with Java Agent.
